docs/source/howto/data.rst

```diff
@@ -260,6 +260,359 @@ Finding data
 Organizing data
 ===============
 
+Grouping sets of nodes together
+-------------------------------
+
+AiiDA's database is great for automatically storing all your data, but sometimes it can be tricky to navigate this flat data store.
+To create some order in this mass of data, you can *group* sets of nodes together, just as you would with files in folders on your filesystem.
+A folder, in this analogy, is represented by the :py:class:`aiida.orm.groups.group.Group` class.
+Each Group instance can hold any amount of nodes and any node can be contained in any number of nodes.
+A typical use case is to store all nodes that share a common property in a single group.
+
+Below we profide a list of typical operations that may be performed with groups.
+
+Create a new Group
+^^^^^^^^^^^^^^^^^^
+From the command line interface:
+
+.. code-block:: console
+
+    $ verdi group create test_group
+
+From the python interface:
```

Comment:
```suggestion
From the Python interface:
```
["Python" should be capitalized if used as a name.](https://github.com/aiidateam/aiida-core/wiki/Writing-documentation#naming-conventions) Please fix also other occurrences.

Reply:
I see a lot of `python` in the other parts of the file, should I change them as well?

Reply:
Yes, please.